authenc.c

```diff
@@ -75,7 +75,7 @@ static int get_userbuf_tls(struct csession *ses, struct kernel_crypt_auth_op *kc
 	// we avoid overflows
 	if ((unsigned long) kcaop->dst_len > (unsigned long) (caop->dst + kcaop->dst_len)) {
 		dwarning(1, "Invalid destination or size");
-                return -EINVAL;
```

Comment:
typically, any fixups should be squashed back onto the original commit that introduced them. This will keep the series of commits clean.